docs/components/board/customlinux.md

```diff
@@ -213,15 +230,104 @@ On your `customlinux` board, create a file in your <file>/home/root</file> direc
 {{% /tab %}}
 {{< /tabs >}}
 
-The following parameters are available for pin mappings:
+The following parameters are available for each pin object:
 
 <!-- prettier-ignore -->
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
-| `name` | string | **Required** | The name of the pin. <br> Example: `"3"`. |
-| `device_name` | string | **Required** | The device name. <br> Example: `"gpiochip0"`. |
-| `line_number` | integer | **Required** | The line number of the pin. <br> Example: `81`. |
-| `pwm_id` | integer | **Required** | The power management id of the pin. Example: `-1`. |
+| `name` | string | **Required** | The name of the pin. This can be anything you want but it is convenient to use the physical board pin number. <br> Example: `"3"`. |
+| `device_name` | string | **Required** | The name of the device in <file>/dev</file> that this pin is attached to. Multiple pins may be attached to the same GPIO chip.  See [GPIO info tips](#tips-for-finding-gpio-information) below. <br> Example: `"gpiochip0"`.
+| `line_number` | integer | **Required** | The line on the chip that is attached to this pin. See [GPIO info tips](#tips-for-finding-gpio-information) below. <br> Example: `81`. |
+| `pwm_chip_sysfs_dir` | string | Optional | Uniquely specifies which PWM device within [sysfs](https://en.wikipedia.org/wiki/Sysfs) this pin is connected to. See [PWM info tips](#tips-for-finding-pwm-information) below. <br> Example: `3290000.pwm`. |
+| `pwm_id` | integer | Optional | The power management ID of the pin. See [PWM info tips](#tips-for-finding-pwm-information) below. <br> Example: `-1`. |
+
+{{% alert title="Tip" color="tip" %}}
+
+`pwm_chip_sysfs_dir` and `pwm_id` only apply to pins with hardware PWM supported and enabled.
+If your board supports hardware PWM, you will need to enable it if it is not enabled by default.
+This process depends on your specific board.
+
+{{% /alert %}}
+
+{{% alert title="Info" color="info" %}}
+
+The current version of `viam-server` creates PWM functionality with software.
+The implementation of hardware-based PWM for custom Linux boards is planned for release in the future, so we recommend that you add PWM information to your board now so that you do not need to update your config later.
```

Comment:
:+1: Again, this is phrased very well! It doesn't draw attention to our software missing this feature, yet it does point that out clearly. Thanks! Sorry again this isn't reliably working already.